admin4.php

```diff
@@ -59,19 +59,19 @@
 				<!-- nav 2 containing the navigation links -->
 				<div class="nav2" style="top: 0vw; position: sticky;">
 					<ul>
-						<li style="margin-left: 0vw; margin-right: 0vw;"><a href="query1.php">Query One</a></li>
-						<li><a href="query2.php">Query Two</a></li>
+						<li class ="hover-underline-animation" style="margin-left: 0vw; margin-right: 0vw;"><a href="query1.php">Query One</a></li>

```

Comment:
Added hover-underline-animation effect class, which has been styled in the style sheet.